packages/common/tsconfig-bases/all-targets.json

```diff
@@ -1,5 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
+  "exclude": ["**/*.test.*", "**/__mocks__/*", "**/__tests__/*"],
```

Comment:
For now I'd exclude what actually exists and needs to be excluded.

Is `__mocks__` directory configurable in Jest? (I remember they had an issue about it.)

Also, I'd personally prefer avoiding `__tests__` directories and instead putting test files right next to actual code. (Test code is as important as production code, not just an after-thought.)

Wondering what others think :thinking:

Reply:
I am fine by adding tests directly next to the files, we should however list all ignore patterns here, otherwise when we'll run the build command these tests could be accidentally added to the build folder.

Reply:
Agreed `__tests__` is not my desired pattern either 🙂

Reply:
If you mean Rollup build for "lib" packages, that always starts at the configured index file, specified in `rollup.config.js` files. Unless that code directly or indirectly references test code, there's no chance test code would be picked up by Rollup bundler.

Reply:
Also, only "lib" packages have `build` script defined; the rest are private/internal helper packages that don't have a build output.